README.md

```diff
@@ -42,25 +40,28 @@ By default, `chef_ingredient` will install using the `packages.chef.io` stable r
 #### Actions
 
 - `install` - (default) Configures the package repository and installs the specified package.
+- `upgrade` - Upgrades the specified package.
 - `uninstall` - Uninstalls the specified package.
 - `remove` - Alias for uninstall
 - `reconfigure` - Performs the `ctl reconfigure` command for the package.
 
 #### Properties
 
-- `product_name`: (name attribute) The product name. See the [PRODUCT_MATRIX.md](https://github.com/chef/mixlib-install/blob/master/PRODUCT_MATRIX.md). For example, `chef-server`, `analytics`, `delivery`, `manage`, etc.
-- `config`: String content that will be added to the configuration file of the given product.
-- `ctl_command`: The "ctl" command, e.g., `chef-server-ctl`. This should be automatically detected by the library helper method `chef_ctl_command`, but may need to be specified if something changes, like a new add-on is made available.
-- `options`: Options passed to the `package` resource used for installation.
-- `version`: Package version to install. Can be specified in various semver-alike ways: `12.0.4`, `12.0.3-rc.3`, and also `:latest`/`'latest'`. Do not use this property when specifying `package_source`. Default is `:latest`, which will install the latest package from the repository.
-- `channel`: Channel to install the products from. It can be `:stable` (default), `:current` or `:unstable`.
-- `package_source`: Full path to a location where the package is located. If present, this file is used for installing the package. Default `nil`.
-- `timeout`: The amount of time (in seconds) to wait to fetch the installer before timing out. Default: default timeout of the Chef package resource - `900` seconds.
-- `accept_license`: A boolean value that specifies if license should be accepted if it is asked for during `reconfigure`action. This option is applicable to only these products: manage, analytics, reporting and compliance. Default: `false`.
-- `platform`: Override the auto-detected platform for which package to install.
-- `platform_version`: Override the auto-detected platform version for which package to install.
-- `architecture`: Override the auto-detected architecture for which package to install.
-- `platform_version_compatibility_mode`: Find closest matching package when platform auto-detection does not find an exact package match in the repository
+| Name  | Type | Default Value  |  Description  |
+|---|---|---|---|
```

Comment:
These are nearly impossible to read in plaintext which is how a lot of users are going to read them. You should expand the table out so that it can be read without github formatting